package/kernel/linux/modules/hwmon.mk

```diff
@@ -97,7 +97,7 @@ define KernelPackage/hwmon-nct6775
   TITLE:=NCT6106D/6775F/6776F/6779D/6791D/6792D/6793D and compatibles monitoring support
   KCONFIG:=CONFIG_SENSORS_NCT6775
   FILES:=$(LINUX_DIR)/drivers/hwmon/nct6775.ko
-  AUTOLOAD:=$(call AutoProbe,nct6775)
+  AUTOLOAD:=$(call AutoLoad,51,nct6775)
```

Comment:
no, there is no dependency, but both drivers claim the same hardware. however, the implementation to support NCT6775 in W83627EHF is rather incomplete, hence the original nct6775 module should be preferred. This can still be achieved with AutoProbe if the w83627ehf driver is patched not to claim the nct677x hardware. Philip is currently testing this, see also
https://git.lede-project.org/?p=lede/dangole/staging.git;a=commit;h=9c01b3dbe96759a710bd24c13adf96b3be188408